### PR TITLE
Simplify code

### PR DESCRIPTION
**Pull request by gaohanghang**
1. Use `Math.max()` and lambda expression simplify some code.
2. Use `Long.parseLong()` replace ` Long.valueOf()`

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@gaohanghang Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=18166)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=18166) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@gaohanghang Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=18166)!

**Comment by snicoll** [comment]
@gaohanghang in order to reduce the noise of notification, please run the build locally and fix violations before submitting rather than relying on our CI to detect them.

**Comment by gaohanghang** [comment]
> @gaohanghang Thank you for the PR. In order to reduce the noise of notification, please run the build locally and fix violations before submitting rather than relying on our CI to detect them.

Sorry, thank you for your suggestion.

File: spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/DeferredLog.java [hunk 1]
```diff
@@ -232,7 +232,6 @@ private static void logTo(Log log, LogLevel level, Object message, Throwable thr
 			return;
 		case FATAL:
 			log.fatal(message, throwable);
-			return;
```
Comment (philwebb) [inline spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/DeferredLog.java]: I'd prefer to leave this in for consistency with the other statements. The other changes look like nice improvements. Thanks!

Comment (gaohanghang) [reply spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/DeferredLog.java]: 👌Thank you for code review.

**Pull request closed without merging**